wikicharts/wikicharts.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #51.    def simple_num_format(value, round_sigfigs=False, sig=2, perc=False, sign=False):
</pre><p>I appreciate the formatting fixes in this function! :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wikimedia-research/movement-metrics/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>